code/game/objects/effects/trackers.dm

```diff
@@ -0,0 +1,96 @@
+
+//object that moves constantly toward a target.
+
+/obj/effect/tracker
+	name = "tracker"
+	unacidable = 1
+	w_type=NOT_RECYCLABLE
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "soul2"
+	mouse_opacity = 0
+	animate_movement = 0
+	var/absolute_X = 0
+	var/absolute_Y = 0
+	var/atom/target = null
+	var/speed = 2
+	var/acceleration = 1
+	var/maxdist = 320
+	var/refresh = 1
+
+/obj/effect/tracker/New()
+	. = ..()
+	absolute_X = (x * 32)
+	absolute_Y = (y * 32)
+
+	spawn(1)
+		process_step()
+
+/obj/effect/tracker/soul
+	name = "soul"
+	icon_state = "soul3"
+
+/obj/effect/tracker/drain
+	name = "blood"
+	color = "red"
```

Comment:
```color = "red"```, color is hexadecimals or matrices, are you seriously telling me it takes particular strings?